README.md

```diff
@@ -88,6 +88,36 @@ Rusty-Lassie's build script copies the DLL into the target directory next to the
 main executable. All you need is to include this DLL in your distribution
 archive.
 
+## Cross-compilation
+
+If you are building your project using
+[Cross](https://github.com/cross-rs/cross), you need to install Go in the Docker
+images used by Cross.
+
+For example, you can add a `Cross.toml` file with the following content to your
+project root:
+
+```toml
+[target.x86_64-unknown-linux-gnu]
+pre-build = [
+  "rm -rf /usr/local/go && curl -L https://go.dev/dl/go1.20.5.linux-amd64.tar.gz | tar -xz -C /usr/local && ln -s /usr/local/go/bin/go /usr/local/bin/ && go version"
+]
+
+[target.aarch64-unknown-linux-gnu]
+pre-build = [
+  "rm -rf /usr/local/go && curl -L https://go.dev/dl/go1.20.5.linux-amd64.tar.gz | tar -xz -C /usr/local && ln -s /usr/local/go/bin/go /usr/local/bin/ && go version"
+]
+```
+
+Cross is always running the container on `amd64`, therefore you must download
+`amd64` version of Go when targeting `aarch64`.
```

Comment:
```suggestion
```

What about instead linking to this project's own Cross.toml? Configuring Cross for Go is out of scope for the Readme IMO

Reply:
+1 to replace this section with a link to Rusty-Lassie's Cross.toml

I think it's important to make it easy for Rusty Lassie consumers to figure out how to compile their projects using Cross.